lib/logstash/outputs/webhdfs.rb

```diff
@@ -0,0 +1,265 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/outputs/base"
+require "stud/buffer"
+
+# Summary: Plugin to send logstash events to to files in HDFS via webhdfs 
+# restapi.  
```

Comment:
Done in https://github.com/dstore-dbap/logstash-output-webhdfs-1/commit/431f8200af1f00e307818cc7eaab2dc31caf51a1